openllm-python/src/openllm/protocol/openai.py

```diff
@@ -138,4 +138,5 @@ def messages_to_prompt(messages: list[Message], model: str, llm_config: openllm_
   for message in messages:
     if message['role'] == 'system': conv_template.set_system_message(message['content'])
     else: conv_template.append_message(message['role'], message['content'])
+  conv_template.append_message('assistant', '')
```

Comment:
What does this do?

Reply:
With the logic of the conversation template, this will add an [/INST] to the end of the prompt to signal that the user message is DONE and the model should return an assistant message.